distribution/docs/src/main/resources/_contents/_tables/Csw_Registry_Store-table-contents.adoc

```diff
@@ -0,0 +1,62 @@
+.[[Csw_Registry_Store]]CSW Registry Store
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
```

Comment:
We have Name and ID here, but these are both being replaced by the URL which pulls the Remote Nodes Identity Name when it connects.